src/main/java/io/jsonwebtoken/lang/JOSEException.java

```diff
@@ -0,0 +1,22 @@
+package io.jsonwebtoken.lang;
```

Comment:
I apologize that this isn't documented anywhere, but the 'lang' package is essentially only for classes that augment/enhance the Java language itself (i.e. things that would have been nice to have in the language).